src/kaleidoscope/driver/keyscanner/ATMega.h

```diff
@@ -0,0 +1,242 @@
+/* -*- mode: c++ -*-
+ * ATMegaKeyScanner -- ATMega-based keyscanner component
```

Comment:
This one should probably just be 'AVR' instead of ATMega. It's reasonably generic